src/ParallelAlgorithms/Interpolation/Protocols/PostInterpolationCallback.hpp

```diff
@@ -0,0 +1,126 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+namespace Parallel {
+template <typename Metavariables>
+struct GlobalCache;
+}  // namespace Parallel
+namespace Tags {
+struct Time;
+}  // namespace Tags
+/// \endcond
+
+namespace intrp::protocols {
+namespace detail {
+struct DummyMetavariables;
+struct DummyTag : db::SimpleTag {
+  using type = int;
+};
+
+template <typename T, typename = std::void_t<>>
+struct has_signature_1 : std::false_type {};
+
+template <typename T>
+struct has_signature_1<
+    T, std::void_t<decltype(T::apply(
+           std::declval<const db::DataBox<DummyTag>&>(),
+           std::declval<const Parallel::GlobalCache<DummyMetavariables>&>(),
+           std::declval<const ::Tags::Time&>()))>> : std::true_type {
```

Comment:
Again, a *tag*?

Reply:
Yes this should just be a `double`, my bad.